src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -225,4 +225,12 @@ public void testPullRequestAnalysisWithNewBlockerAndCriticalIssues() {
 
     verify(pullRequestFacade).createOrUpdateSonarQubeStatus(GHCommitState.ERROR, "SonarQube reported 2 issues, with 1 critical and 1 blocker");
   }
+
+  @Test
+  public void should_update_sonarqube_status_even_if_unexpected_errors_were_raised() {
+    // raises an NPE due to improper mock setup; not realistic, but good enough for this test
+    pullRequestIssuePostJob.execute(context);
+
+    verify(pullRequestFacade).createOrUpdateSonarQubeStatus(GHCommitState.ERROR, "unexpected error occurred while posting comments");
```

Comment:
I'd suggest to leverage the `details` tag for extra details:

<details>
  <summary>Click for error details</summary>
  whatever
</details>

---------------------
source markdown / html:
```
<details>
  <summary>Click for error details</summary>
  whatever
</details>
```